baseplate/experiments/providers/__init__.py

```diff
@@ -87,11 +94,12 @@ def parse_experiment(config):
         )
         version = None
     now = time.time()
-    if now < start_ts or now > stop_ts:
-        return ForcedVariantExperiment(None)
 
     enabled = config.get("enabled", True)
-    if not enabled:
+    if now < start_ts or now > stop_ts:
```

Comment:
either or both of start_ts and stop_ts could be None, I think, so this might raise.

Reply:
if start_ts or stop_ts is None, there's an error that would be thrown a couple lines above (line 83).

Though, now that you mention it, if they're explicitly passed as None, that'll cause an issue. Fix inbound.